tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -86,6 +86,51 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-efx-generic-4ch.bin,\
 DEEPBUFFER_FW_DMA_MS=100,EFX_FIR_PARAMS=passthrough,EFX_IIR_PARAMS=passthrough,\
 EFX_DRC_COMPONENT=multiband,EFX_MBDRC_PARAMS=passthrough"
 
+# With 16 kHz DMIC1
```

Comment:
One question:  can we have a flag to build these DMIC1 ?  Currently we do use these DMIC1 tplgs but need to add burden for CI

Reply:
@RanderWang @fredoh9 @mengdonglin We had a testing gap and missed that alsatplg did not work correctly for two DMIC DAI case. I'd like to add some cases with DMIC1 and additionally with other than default 16 kHz to increase coverage. Suggestions of which platforms to use and what kind of topologies set to build are welcome. I used here two platforms I had hands-on access to. A MTL RVP platform and LNL might make better sense. What do you think?

Reply:
Yes, MTL RVP is preferred. SDW platforms use SDW codec DMIC in current situation, do we need to test with it ?


Please check this in HDA machine driver,  does your PR work with this ?
````c
	{
		.name = "dmic16k",
		.id = 7,
		.dpcm_capture = 1,
		.no_pcm = 1,
		SND_SOC_DAILINK_REG(dmic16k, dmic_codec, platform),
	},
````